docs/docs/guides/application-development.md

```diff
@@ -53,58 +48,129 @@ my-app/
 
 To run dependent services locally you can use the included `docker-compose.yaml` file. This file includes databases, DMSS service, and job service. 
 
-Start services by running:
+Start services in the background (-d detached mode) by running:
 
-```
-docker compose up
+```bash
+docker-compose pull
+docker compose up --build -d
 ```
 
 To start the app development server, run:
 
-```
+```bash
 npm start
 ```
 
-This will open the app at [localhost:3000](http://localhost:3000).
+The web app can now be reached at [http://localhost:3000](http://localhost:3000) in the web browser. The web page will automatically reload if you make changes to the code.
+You will see the build errors and lint warnings in the console.
+
+:::note 
+
+Remember: You must have the docker-compose services running to use the web application.
+
+:::
+
+## Reset data
+
+We recommend create a virtual Python environment. 
+
+Create a new virtual environment by running: 
+
+```bash
+python3 -m venv .venv
+``` 
 
-To populate DMSS, run:
+Activate the virtual environment by running:
```

Comment:
Perhaps remove all this python/venv stuff?
Will propably not apply to many environments (like windows in Equinor) anyway.
Just shorten in to "Install the python package dm-cli"

Reply:
I made it a tip (in a accordion), so that it is not that visible 👍